internal/k8s/reconciler/route_test.go

```diff
@@ -50,36 +42,28 @@ func TestTCPRouteID(t *testing.T) {
 func TestRouteCommonRouteSpec(t *testing.T) {
 	t.Parallel()
 
-	config := K8sRouteConfig{
-		Logger: hclog.NewNullLogger(),
-	}
-
 	expected := gwv1alpha2.CommonRouteSpec{
 		ParentRefs: []gwv1alpha2.ParentReference{{
 			Name: "expected",
 		}},
 	}
 
-	require.Equal(t, expected, newK8sRoute(&gwv1alpha2.HTTPRoute{
+	assert.Equal(t, expected, newK8sRoute(&gwv1alpha2.HTTPRoute{
```

Comment:
These are just improving our test hygiene. If one case fails, I still want to see that the other cases pass/fail as well instead of having `require` stop the whole test